### DISCUSSION TITLE
Type error with load function

### DISCUSSION THREAD

**Discussion by Lavanya-Bala** [Eomys/pyleecan#365] [Q&A]
Hi,

I am trying to run the optimisation from the tutorial and the following error occurs. 
As far as I can guess, this is a backend issue? Could you please let me know if there is a way to go about this ? 

Traceback (most recent call last):
  File "\site-packages\pyleecan\Functions\Load\import_class.py", line 8, in import_class
    module = __import__(mod_name + "." + class_name, fromlist=[class_name])
TypeError: can only concatenate str (not "bytes") to str

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "", line 312, in <module>
    res = load("opti_out.h5")
  File "\site-packages\pyleecan\Functions\load.py", line 111, in load
    return init_data(init_dict, file_path)
  File "\site-packages\pyleecan\Functions\load.py", line 51, in init_data
    class_obj = import_class("pyleecan.Classes", obj.get("__class__"), "")
  File "\site-packages\pyleecan\Functions\Load\import_class.py", line 20, in import_class
    "Unknow class name " + class_name + " when loading"
TypeError: can only concatenate str (not "bytes") to str

**Comment by BonneelP** [comment]
Hello,

What a coincidence, I'm working on this tonight (and maybe tomorrow as well) so I have seen this error message too many times: the issue here is the version of h5py. I expect that you are using a version >3 and the current requirements of pyleecan is h5py==2.10.0.  Reverting your version of the package to 2.10 should solve your problem. 
I currently working on making pyleecan work with the latest version of h5py so hopefully it shouldn't happen again.

Best regards,
Pierre

**Reply by Lavanya-Bala** [reply]
That worked, thank you!
